src/Angor/Client/Pages/Signatures.razor

```diff
@@ -589,6 +599,27 @@
             });
     }
 
+    Dictionary<string, string> pubkeyWithMessages = new();
+
+    protected async Task ScanForDmMessages()
+    {
+        foreach (var signatureRequest in signaturesRequests)
```

Comment:
There is a message type called count in nostr that you send the same filter and it returns how many events are there for it

Reply:
Good question, perhaps we can implement something based on support for each relay in the nostr-client